pkg/config/config.go

```diff
@@ -213,6 +214,24 @@ func New(v *viper.Viper, co *Options) (*Config, error) {
 
 			backend = backends.NewYandexCloudLockboxBackend(sdk.LockboxPayload().Payload())
 		}
+	case types.OnePasswordConnect:
+		{
+			if !v.IsSet(types.EnvOPConnectToken) ||
```

Comment:
Wondering if this is necessary, assuming that the `OP_CONNECT` vars are the ones expected by the SDK; omitting this check and deferring to the SDK means we can support any auth methods this backend supports going forward with no code changes, as long as we set all `OP_CONNECT` prefixed config keys in the environment (which your change to add `op_connect` to the backend prefixes slice does). Not a big deal, just a thought